src/sbvr-api/sbvr-utils.coffee

```diff
@@ -559,6 +559,11 @@ exports.runRule = do ->
 					return result
 		.nodeify(callback)
 
+exports.getAPIClient = (vocab) ->
```

Comment:
You can just use `sbvrUtils.api[vocab]` directly (it's what everything else does), there's no need for this function